mappings/net/minecraft/item/ItemConvertible.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS bhg net/minecraft/item/ItemConvertible
```

Comment:
Like the method, the interface could potentially be named `AsItem` too

Reply:
hmm, does this match the naming convention? Any other class named similarly in yarn already?

Reply:
I don't think `AsItem` is a very descriptive name. I think `ItemConvertible` is fine.